CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to a Rearc AWS Data Exchange product
+
+🎉🥳 First of all, **THANK YOU** for being interested in contributing to one of our projects 🎉🥳
+
+From the start of our AWS Data Exchange (ADX) initiative we knew we wanted to have an open dialogue with our subscribers. While we try our best to optimize the packaging of our ADX products to be as accommodating as possible, we appreciate there may be use-cases we have overlooked. By contributing to this project, you have an opportunity to improve the usability of the data included with this product not only for yourself, but for anyone who subscribes to this product through ADX. If your contribution is applicable to more than one of our ADX listings, we will gladly improve those other products as well!
+
+You can learn about Rearc’s data services by visiting [https://www.rearc.io/data]( https://www.rearc.io/data).
```

Comment:
I don't think we need to drive people in our CONTRIBUTORS.md to that (mostly marketing at this point) website.

Reply:
Removed sentence in line 7 driving users to the Rearc Data website, left it under "Additional Resources" at the bottom of the document